ngrinder-frontend/src/js/components/agent/List.vue

```diff
@@ -175,12 +230,24 @@
                 this.selectedAgents = [];
             }
         }
+
+        get ids() {
+            return { params: { ids: this.selectedAgents.join(',') } };
+        }
```

Comment:
It does not need to be a computed property. Just `getParams() { ... }` looks better.